src/main/java/uk/gov/hmcts/reform/prl/models/dto/ccd/CaseData.java

```diff
@@ -735,6 +735,17 @@ public CaseData setIssueDate() {
     private ResponseDocuments respondentDc8;
     private ResponseDocuments respondentEc8;
 
+    @JsonProperty("respondentAc8Documents")
+    private List<Element<ResponseDocuments>> respondentAc8Documents;
+    @JsonProperty("respondentBc8Documents")
+    private List<Element<ResponseDocuments>> respondentBc8Documents;
+    @JsonProperty("respondentCc8Documents")
+    private List<Element<ResponseDocuments>> respondentCc8Documents;
+    @JsonProperty("respondentDc8Documents")
+    private List<Element<ResponseDocuments>> respondentDc8Documents;
+    @JsonProperty("respondentEc8Documents")
+    private List<Element<ResponseDocuments>> respondentEc8Documents;
```

Comment:
this change will impact respondent solicitor and respondent citizen journey. Also it will need data migration due to clean up activity.

It must be taken up in IP sprint at a later stage